system-variables.md

```diff
@@ -269,6 +275,18 @@ mysql> SELECT * FROM t1;
 >
 > Unlike in MySQL, the `max_execution_time` system variable currently works on all kinds of statements in TiDB, not only restricted to the `SELECT` statement. The precision of the timeout value is roughly 100ms. This means the statement might not be terminated in accurate milliseconds as you specify.
 
+### plugin_dir
```

Comment:
@TomShawn The scope and description of `plugin_dir` is not consistent with https://github.com/pingcap/docs-cn/pull/11495